FrameworkSystem/scripts/dirac-admin-update-instance.py

```diff
@@ -179,12 +184,13 @@ def restartHost(hostName):
   return result
 
 
-def updateInstance(version, hosts):
+def updateInstance(version, hosts, retry):
   """
   Update each server of an instance and restart them
 
   :param version: version vArBpC you want to update to
   :param hosts: list of hosts to be updated
+  :param retry: number of retry attempts on hosts that have failed to update
```

Comment:
```suggestion
  :param int retry: number of retry attempts on hosts that have failed to update
```
(also add the type also for the other parameters, also in other functions)